features/270-next-skycli/commands.md

```diff
@@ -257,6 +257,32 @@ $ skycli app view-user-config > ./user-config.yaml
 $ skycli app update-user-config -f ./user-config.yaml
 ```
 
+### skycli app get-k8s-credentials
+
+`skycli app get-k8s-credentials`
+
+Fetch the credentials to access k8s resources with kubectl.
+The credentials are written to the machine's kubeconfig.
+
+The name of the context is `skygear-<app-name>`.
```

Comment:
Since we have different clusters (e.g. staging or production...etc), maybe we should update the context name so that developer can distinguish which cluster they are using?